packages/host/app/components/operator-mode/container.gts

```diff
@@ -932,7 +960,7 @@ export default class OperatorModeContainer extends Component<Signature> {
         position: absolute;
         top: 0;
         left: 0;
-
+        z-index: 2;
```

Comment:
This is probably not needed yet but we may eventually want to extract these into named layers if the layering gets too messy.

Reply:
Yeah that could make things more manageable. We'll see